pkgs/applications/misc/qMasterPassword/default.nix

```diff
@@ -1,18 +1,18 @@
-{ lib, stdenv, mkDerivation, fetchFromGitHub, qtbase, qmake, libX11, libXtst, openssl, libscrypt }:
+{ lib, stdenv, mkDerivation, fetchFromGitHub, qtbase, qmake, qttools, libX11, libXtst, openssl, libscrypt }:
 
 mkDerivation rec {
-  name = "qMasterPassword";
-  version = "1.2.2";
+  pname = "qMasterPassword-unstable";
+  version = "2022-01-28";
 
   src = fetchFromGitHub {
     owner = "bkueng";
-    repo = name;
-    rev = "v${version}";
-    sha256 = "0l0jarvfdc69rcjl2wa0ixq8gp3fmjsy9n84m38sxf3n9j2bh13c";
+    repo = "qMasterPassword";
+    rev = "master";
```

Comment:
I missed this on the first pass, but needs to be pinned to a specific commit, or the build will break when upstream changes.

Reply:
Done